site/scripts/get-contributors.js

```diff
@@ -41,9 +39,29 @@ async function main() {
 		sprite.composite(image, i * SIZE, 0);
 	}
 
-	await sprite.quality(80).write(`../static/contributors.jpg`);
-	// TODO: Optimizing the static/contributors.jpg image should probably get automated as well
-	console.log('remember to additionally optimize the resulting /static/contributors.jpg image file via e.g. https://squoosh.app ');
+	await sprite.quality(80).getBuffer(Jimp.MIME_JPEG, (err, buffer) => {
+		let quality = 75;
+
+		let content = `--xxxxxxxxxx\r\nContent-Disposition: form-data; name="qlty"; \r\n\r\n${quality}\r\n--xxxxxxxxxx\r\nContent-Disposition: form-data; name="files"; filename="contributors.jpg"\r\nContent-Type:application/octet-stream\r\n\r\n`;
+		let end = `\r\n--xxxxxxxxxx--\r\n`;
+
+		let body = Buffer.concat([Buffer.from(content, "utf8"), new Buffer(buffer, "binary"), Buffer.from(end)]);
+
+		fetch('https://api.resmush.it/ws.php', {
+		    method: 'POST',
+		    headers: {
+		        "Content-type": 'multipart/form-data; boundary=xxxxxxxxxx',
+		    },
+		    body: body
+		}).then(async (res) => {
```

Comment:
This is a mix of `then` and `await`. I'd prefer it to be `await` everywhere. Apart from that this looks good to me; but I didn't do this by hand previously so can't tell if this is semanticall the same.

Reply:
@dummdidumm Just switched everything over to `await`